x-pack/elastic-agent/pkg/core/server/server.go

```diff
@@ -148,6 +153,7 @@ func (s *Server) Start() error {
 	if ok := certPool.AppendCertsFromPEM(s.ca.Crt()); !ok {
 		return errors.New("failed to append root CA", errors.TypeSecurity)
 	}
+	//nolint:gosec // G402: TLS MinVersion too low.
```

Comment:
Not sure if it breaks backward compatibility but it would be nice if we stopped accepting insecure TLS versions... Although this also silences the linter.

@cmacknz  maybe we want to address this in a separate issue?